exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/TransferTxHistoryTest.java

```diff
@@ -49,14 +47,20 @@
     LibraryLoader.load();
   }
 
-  private static final PublicKey ACCOUNT_1 = PredefinedOwnerKeys.firstOwnerKey;
-  private static final PublicKey ACCOUNT_2 = PredefinedOwnerKeys.secondOwnerKey;
+  private static final PublicKey ACCOUNT_1 = PredefinedOwnerKeys.FIRST_OWNER_KEY;
+  private static final PublicKey ACCOUNT_2 = PredefinedOwnerKeys.SECOND_OWNER_KEY;
 
   @Test
   void transfersHistoryBetweenTwoAccountsTest() throws Exception {
     try (Database db = MemoryDb.newInstance();
         Cleaner cleaner = new Cleaner()) {
       Fork view = db.createFork(cleaner);
+      HashCode hash = HashCode.fromString("a0a0a0a0");
+      TransactionContext context = TransactionContext.builder()
```

Comment:
It is generally not OK to reuse a context for two different transactions:
      (1) The hash is the same, but two different txs can't have the same hash.
      (2) The signer is the same, which, again, with two different source accounts is not possible.